test/language.js

```diff
@@ -58,23 +58,23 @@ describe('language()', () => {
         expect(language).to.equal('');
     });
 
-    it('returns first preference if header has *', () => {
+    it('returns first preference if header has * and is unmatched', () => {
 
-        const language = Accept.language('da, en-GB, en, *', ['en-US']);
+        const language = Accept.language('da, en-GB, *', ['en-US']);
         expect(language).to.equal('en-US');
     });
 
     it('returns first found preference that header includes', () => {
 
         const language = Accept.language('da, en-GB, en', ['en-US', 'en-GB']);
-        expect(language).to.equal('en-GB');
+        expect(language).to.equal('en-US');
```

Comment:
With range matching, `en-US` should be returned as the highest ordered preference when selecting from equally weighted options.